src/elexmodel/client.py

```diff
@@ -91,9 +91,11 @@ def _check_input_parameters(
             raise ValueError("beta is not valid. Has to be either an integer or a float.")
         if not isinstance(robust, bool):
             raise ValueError("robust is not valid. Has to be a boolean.")
-        if not isinstance(lambda_, (float, int)):
-            raise ValueError("lambda is not valid. It has to be numeric.")
-        if lambda_ < 0:
+        if not isinstance(lambda_, list):
+            raise ValueError("lambda is not valid. It has to be a list.")
+        if not all(isinstance(e, (int, float)) for e in lambda_) and len(lambda_) > 0:
+            raise ValueError("lambda is not valid. It has to be a list of numbers.")
+        if len(lambda_) < 0:
```

Comment:
I'm not 100% sure I understand how this could happen?

Reply:
I put this check in just in case there's a typo while inputting the lambda values- is that fine? I switched the order of the two checks on that line like you mentioned in a comment below